modules/launchd/default.nix

```diff
@@ -80,11 +72,11 @@ let
       };
 
       config = {
-        command = mkIf (config.script != "") (pkgs.writeScript "${name}-start" ''
+        command = mkIf (config.script != "") [ (pkgs.writeScript "${name}-start" ''
```

Comment:
Should we add a `wait4path` in here? (I’m fine if you’d rather leave that to future work.)

Reply:
I don't think it's necessary as `serviceConfig.ProgramArguments` already does `wait4path /nix/store`